### DISCUSSION TITLE
error from daemon in stream: Error grabbing logs: invalid character '\x00' looking for beginning of value

### DISCUSSION THREAD

**Discussion by sydfernandes** [open-webui/open-webui#5177] [General]
error from daemon in stream: Error grabbing logs: invalid character '\x00' looking for beginning of value

What is this about?

**Comment by justinh-rahb** [comment]
https://github.com/docker/for-linux/issues/140

Your Docker host likely has corrupted logs, could be your filesystem got corrupted from a sudden power loss, or your physical media is degraded.
